sos/report/plugins/hpilorest.py

```diff
@@ -9,32 +9,21 @@
 from sos.report.plugins import Plugin, IndependentPlugin, PluginOpt
 
 
-class Hpssm(Plugin, IndependentPlugin):
+class Hpilorest(Plugin, IndependentPlugin):
 
-    short_desc = 'HP Smart Storage Management'
+    short_desc = 'HP RESTful Interface Tool'
 
-    plugin_name = 'hpssm'
+    plugin_name = 'hpilorest'
     profiles = ('system', 'storage', 'hardware',)
-    packages = ('ilorest', 'ssacli', 'ssaducli',)
+    packages = ('ilorest',)
 
     option_list = [
         PluginOpt('debug', default=False, desc='capture debug data')
     ]
```

Comment:
Agreed, this is why I mentioned it above. The other thought I had is if this was only collected by setting the debug option true, because the original plugin was enabled by default for `ssacli` collection, would be to remove the debug altogether, but also set the `ilorest` plugin default to disabled. This would continue to mimic some of the behavior of not collecting `ilorest` unless requested.

I'm open to any of the methods here honestly.

- Leaving it it in `hpssa` with debug
- Remove debug from `ilorest` and
  - Plugin defaults to enabled and collects command output
  - Plugin defaults to disabled, requiring activation to collect output.

Reply:
Let's leave it in `hpssa` with the debug option. After thinking on it a bit more, I think it might be seen as a bit heavy handed for a default collection.

Reply:
I thought through the adjustments, and moving ilorest back into the "main" plugin and leaving debug enabled, it seemed to make more sense to refactor and reduce the changes down.

- Left the original hpssm plugin/file name.
- Updated the section for controller enumeration and gathering details from each with ssacli.
- Added an accurate (although somewhat verbose) docstring
- Updated the style of all `add_cmd_output()` as requested
- Updated the `ilorest` output gathering to use the `--directorypath` flag. I suspect `runat` will handle it but this ensures the output is where it is expected if the working directory is no longer the default output path.